medocker/docker_shared/README

```diff
@@ -0,0 +1,22 @@
+To compile and upload:
+- Create a directory with this structure:
+├── WarpX
+│   ├── medocker
+│   │   ├── docker_<yourimagefolder>
+│   │   │   ├── Dockerfile
+│   │   │   ├── Makefile.local
+│   │   │   ├── Makefile.local.pympi
+│   │   │   └── setup.local.py
+│   │   ├── docker_shared
+│   │   │   ├── make_docker.sh
+│   │   │   ├── README
+│   │   │   └── run_warp.sh
+│   ├── mewarpx
+├── warpx-data
+
+- Move into the docker_<yourimagefolder>
+- Make sure the aws command line is installed and your credentials are
+  configured.
+- Run ../docker_shared/make_docker.sh <current git branch or tag> <yourimagesuffix>
+  (eg with a suffix '-ubuntumpich', and a git branch 'psdevel', this will push
+   to simteam/metools-psdevel-ubuntumpich)
```

Comment:
```suggestion
To compile and upload:
- Create a directory with this structure:
├── WarpX
│   ├── medocker
│   │   ├── docker_<yourimagefolder>
│   │   │   └── Dockerfile
│   │   ├── docker_shared
│   │   │   ├── make_docker.sh
│   │   │   ├── README
│   │   │   └── run_warpx.sh
│   ├── mewarpx
├── warpx-data

- Move into the docker_<yourimagefolder>
- Make sure the aws command line is installed and your credentials are
  configured.
- Run ../docker_shared/make_docker.sh <current git branch or tag> <yourimagesuffix>
  (eg with a suffix '-ubuntumpich', and a git branch 'psdevel', this will push
   to simteam/mewarpx-psdevel-ubuntumpich)
```